python/cudf/cudf/core/dataframe.py

```diff
@@ -5857,7 +5854,7 @@ def _reduce(
             )
             source = self._get_columns_by_label(numeric_cols)
             if source.empty:
-                return Series(index=cudf.StringIndex([]))
+                return Series(index=self.index)
```

Comment:
Is this a pandas 2.0 change? You get an empty output but now with a nonempty index?

Reply:
Yup, pandas 2.0 fixed an issue where the index was being dropped for empty outputs.